code/datums/mind.dm

```diff
@@ -65,15 +65,14 @@
 	var/force_escaped = FALSE  // Set by Into The Sunset command of the shuttle manipulator
 	var/list/learned_recipes //List of learned recipe TYPES.
 
-	/// Our skill holder.
-	var/datum/skill_holder/skill_holder
+	/// Skill datum
+	var/datum/skill_list_bay/bay_skills = new /datum/skill_list_bay
```

Comment:
Initiating this on instantiation is probably better.
You don't need to use the full typepath again if the variable already has it, just `new()` will suffice.
It might be better if we use a singleton system similar to the armor datums, though.

Reply:
Singleton might not work, I think, since skills update in-game.

Reply:
Armors update as well. You just swap from one singleton to another, and keep them indexed.